backend/src/libs/packages/s3/libs/types/types.ts

```diff
@@ -0,0 +1 @@
+export { type AWSFileUploadRequestDto } from 'shared/build/index.js';
```

Comment:
```suggetion
export { type FileUploadRequestDto } from 'shared/build/index.js';
```

Reply:
we have the exact same type name already - so I'll use S3FileUploadRequestDto